pkg/secretlock/local/masterkey_reader.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package local
+
+import (
+	"fmt"
+	"io"
+)
+
+// newReader will create a new master key reader using passphrase
+func newReader(passphrase string, encMasterKey []byte) (*Reader, error) {
```

Comment:
nit pick - you could just return `io.Reader`

Reply:
good idea

Reply:
the reader file has been removed.. there is no more reader implementation in this change.